mergify_engine/tests/functional/actions/test_comment.py

```diff
@@ -20,6 +20,8 @@
 
 
 class TestCommentAction(base.FunctionalTestBase):
+    SUBSCRIPTION_ACTIVE = True
```

Comment:
Can you only set this in a different test class so we're able to test both scenario, with and without sub enabled?

```python
class TestCommentActionWithSub(base.FunctionalTestBase):
    SUBSCRIPTION_ACTIVE = True

     def test_comment_with_oauth_token(self):
         [...]
```

and add a test with no sub enabled 😄